src/Development/IDE/Types/Action.hs

```diff
@@ -0,0 +1,69 @@
+module Development.IDE.Types.Action
+  (DelayedAction(..)
+  ,DelayedActionInternal
+  ,ActionQueue
+  ,newQueue
+  ,pushQueue
+  ,popQueue
+  ,doneQueue
+  ,peekInProgress
```

Comment:
`  , peekInProgress` 

space

Reply:
I am layout and spacing blind! Let me run ormolu